.github/workflows/examples.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       run: |
         source ./dev/install-common-deps.sh
         source activate test-environment
-        pytest --verbose tests/examples --large
+        pytest --verbose tests/examples --durations=30 --large
```

Comment:
I had [this comment](https://github.com/mlflow/mlflow/pull/3266#pullrequestreview-466455798) in #3266, if we can avoid pinning down a particular number we do not need to keep updating it.